text_classifier/sigopt_creds.py

```diff
@@ -0,0 +1,4 @@
+# Find your creds at https://sigopt.com/user/profile
```

Comment:
yeah basically a utility that takes in all of the credentials and spits out a file (I always like things like that, because it's on less comma or quotation mark to worry about)